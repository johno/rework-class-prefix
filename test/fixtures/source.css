.my-class {
  color: green;
}

fieldset[disabled] .my-class {
  color: tomato;
}

.my-class .with-another-class {
  box-sizing: border-box;
}

@media screen and (min-width: 48rem) {
  .my-class {
    color: pink;
  }
}
