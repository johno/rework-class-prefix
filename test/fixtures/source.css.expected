.prfx-my-class {
  color: green;
}

fieldset[disabled] .prfx-my-class {
  color: tomato;
}

.prfx-my-class .prfx-with-another-class {
  box-sizing: border-box;
}

@media screen and (min-width: 48rem) {
  .prfx-my-class {
    color: pink;
  }
}
